### PR TITLE
Don't need has_post_thumbnail :(

### PR DESCRIPTION
**Pull request by mayukojpn**

### CODE REVIEW

**Comment by mayukojpn** [comment]
I'm sorry! 💦

File: wp-postviews.php [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <?php
 /*
-Plugin Name: WP-PostViews
+Plugin Name: WP-PostViews-thumbnail
```
Comment (lesterchan) [inline wp-postviews.php]: I think this change is accidental, can you fix it?

**Comment by lesterchan** [comment]
@mayukojpn just one more small fix and I will merge it =D

File: wp-postviews.php (hunk 1 above)
Comment (mayukojpn) [inline wp-postviews.php]: Yes I did 😱 sorry again!

**Comment by lesterchan** [comment]
lol Thanks!

**Pull request merged**